scripts/bootswatch.js

```diff
@@ -7,7 +7,9 @@ const path    = require('path');
 const fs      = require('fs');
 const request = require('request');
 
-const version = process.argv[2];
+const version  = process.argv[2];
+const verMajor = version[0];
```

Comment:
Can't we just do this programmatically instead of adding a new arg?

Reply:
It's not a new arg, it's taking the first character of the string.

Reply:
True, sorry, too tired :/